From 29e0aafc23624a2fc8dfc114f1ca52cf46484f86 Mon Sep 17 00:00:00 2001
Message-ID: <29e0aafc23624a2fc8dfc114f1ca52cf46484f86.1711239854.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1711239854.git.sharlatanus@gmail.com>
References: <cover.1711239854.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 22 Mar 2024 21:17:34 +0000
Subject: [PATCH 03/12] gnu: go-github-com-alecthomas-template: Move to
 golang-xyz.

* gnu/packages/golang.scm (go-github-com-alecthomas-template): Move from
here ...
* gnu/packages/golang-xyz.scm: ... to here.

Change-Id: Iffdd88439819c526cd6dd940192bf48995bf76bc
---
 gnu/packages/golang-xyz.scm | 27 +++++++++++++++++++++++++++
 gnu/packages/golang.scm     | 27 ---------------------------
 2 files changed, 27 insertions(+), 27 deletions(-)

diff --git a/gnu/packages/golang-xyz.scm b/gnu/packages/golang-xyz.scm
index fb3c31a38e..bd808d44c3 100644
--- a/gnu/packages/golang-xyz.scm
+++ b/gnu/packages/golang-xyz.scm
@@ -200,6 +200,33 @@ (define-public go-github-com-alecthomas-participle-v2
 \"unmarshalling\" an instance of a grammar into a struct.")
     (license license:expat)))
 
+(define-public go-github-com-alecthomas-template
+  (let ((commit "a0175ee3bccc567396460bf5acd36800cb10c49c")
+        (revision "0"))
+    (package
+      (name "go-github-com-alecthomas-template")
+      (version (git-version "0.0.0" revision commit))
+      (source
+       (origin
+         (method git-fetch)
+         (uri (git-reference
+               (url "https://github.com/alecthomas/template")
+               (commit commit)))
+         (file-name (git-file-name name version))
+         (sha256
+          (base32 "0qjgvvh26vk1cyfq9fadyhfgdj36f1iapbmr5xp6zqipldz8ffxj"))))
+      (build-system go-build-system)
+      (arguments
+       (list
+        #:import-path "github.com/alecthomas/template"))
+      (home-page "https://github.com/alecthomas/template")
+      (synopsis "Fork of Go's text/template adding newline elision")
+      (description
+       "This is a fork of Go 1.4's text/template package with one addition: a
+backslash immediately after a closing delimiter will delete all subsequent
+newlines until a non-newline.")
+      (license license:bsd-3))))
+
 (define-public go-github-com-anmitsu-go-shlex
   (package
     (name "go-github-com-anmitsu-go-shlex")
diff --git a/gnu/packages/golang.scm b/gnu/packages/golang.scm
index 062985c0ce..18f0e6f7ac 100644
--- a/gnu/packages/golang.scm
+++ b/gnu/packages/golang.scm
@@ -9609,33 +9609,6 @@ (define-public go-github-com-skratchdot-open-golang
       (home-page "https://github.com/skratchdot/open-golang")
       (license license:expat))))
 
-(define-public go-github-com-alecthomas-template
-  (let ((commit "a0175ee3bccc567396460bf5acd36800cb10c49c")
-        (revision "0"))
-    (package
-      (name "go-github-com-alecthomas-template")
-      (version (git-version "0.0.0" revision commit))
-      (source (origin
-                (method git-fetch)
-                (uri (git-reference
-                      (url "https://github.com/alecthomas/template")
-                      (commit commit)))
-                (file-name (git-file-name name version))
-                (sha256
-                 (base32
-                  "0qjgvvh26vk1cyfq9fadyhfgdj36f1iapbmr5xp6zqipldz8ffxj"))))
-      (build-system go-build-system)
-      (arguments
-       '(#:import-path "github.com/alecthomas/template"
-         #:phases %standard-phases))
-      (synopsis "Fork of Go's text/template adding newline elision")
-      (description
-       "This is a fork of Go 1.4's text/template package with one addition: a
-backslash immediately after a closing delimiter will delete all subsequent
-newlines until a non-newline.")
-      (home-page "https://github.com/alecthomas/template")
-      (license license:bsd-3))))
-
 (define-public go-github-com-alecthomas-units
   (let ((commit "2efee857e7cfd4f3d0138cc3cbb1b4966962b93a")
         (revision "0"))
-- 
2.41.0

